### PR TITLE
Busconf

### PR DESCRIPTION
**Pull request by MiyamotoAkira**
A write up about busconf

### CODE REVIEW

**Comment by MiyamotoAkira** [comment]
DO NOT MERGE YET. Want to check with the people from busconf

File: _posts/2018-08-07-busconf-2018.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+I just attended [Busconf](http://www.bus-conf.org/index.html), which is a Functional Programming Unconference in its second year. And the experience has been greatly satisfying.
+
+### First, 
+the location, at the [SeminarZentrum](https://www.natuerlich-tagen.de/) in Rückersbach, a bit outside of Frankfurt, is an incredible and beautiful choice (if you want to have a view through google maps you can see it [here](https://www.google.com/maps/@50.0437687,9.1096571,3a,89.6y,323.65h,102.32t/data=!3m8!1e1!3m6!1sAF1QipPlVnBMK25MFA_-Sx1xjU050WdCOwQ-_bd69C4-!2e10!3e11!6shttps:%2F%2Flh5.googleusercontent.com%2Fp%2FAF1QipPlVnBMK25MFA_-Sx1xjU050WdCOwQ-_bd69C4-%3Dw203-h100-k-no-pi-0-ya167.02386-ro0-fo100!7i9728!8i4080)). I was even surprised that taxis would go over there. But to achieve a sense of community, is just perfect. There is nothing else to do but to mingle with the rest of the attendees. And I leave having met lots of interesting people.
```
Comment (Gryff) [inline _posts/2018-08-07-busconf-2018.md]: But to achieve a sense of community, *it* is just perfect.

File: _posts/2018-08-07-busconf-2018.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+### Second,
+the talks where very informative and enjoyable:
```
Comment (Gryff) [inline _posts/2018-08-07-busconf-2018.md]: the talks *were* very informative

File: _posts/2018-08-07-busconf-2018.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+[Tom Harding](https://github.com/i-am-tom) and [Justin Woo](https://github.com/justinwoo) did some talks around Purescript that actually made want to do some frontend (I am one of those people that don't like much neither JS the language nor the ecosystem). 
+
+[Christopher Welcz](https://twitter.com/ChristophWelcz) did a fantastic work with his presentation of functional programming in Java using Vavr. I have used that library before, and he has actually taken away some of my misgivings about it.
```
Comment (Gryff) [inline _posts/2018-08-07-busconf-2018.md]: Maybe "did a fantastic *job* with his presentation" ?

File: _posts/2018-08-07-busconf-2018.md (hunk 2 above)
Comment (MiyamotoAkira) [reply _posts/2018-08-07-busconf-2018.md]: Done

File: _posts/2018-08-07-busconf-2018.md (hunk 3 above)
Comment (MiyamotoAkira) [reply _posts/2018-08-07-busconf-2018.md]: Done

File: _posts/2018-08-07-busconf-2018.md (hunk 1 above)
Comment (MiyamotoAkira) [reply _posts/2018-08-07-busconf-2018.md]: Done

**Pull request merged**